packages/turbo-codemod/package.json

```diff
@@ -17,41 +17,46 @@
     "build": "tsup",
     "test": "jest",
     "lint": "eslint src/**/*.ts",
-    "check-types": "tsc --noEmit"
+    "check-types": "tsc --noEmit",
+    "add-transformer": "plop"
   },
   "dependencies": {
+    "axios": "0.27.2",
     "chalk": "2.4.2",
-    "execa": "5.1.1",
+    "commander": "^9.5.0",
```

Comment:
create-turbo uses `meow`. Do we want to either use commander there or use meow here?

Reply:
Yea good callout. `meow` is _super_ light weight. Which is great for very basic CLI's (single command). But once you get more than a single command and need nested help etc, `meow` gets to be pretty verbose. Commander has a nice API, handles a lot of this for you, and is zero dependencies so it's a good choice. 

I think when we revisit create-turbo we could consider using it there as well.